README.md

```diff
@@ -140,12 +140,20 @@ The pulsar-manager can monitor topics and subscriptions.
 
 3. Access Pulsar manager website.
 
-    Use the account and the password to log in to the Pulsar manager website as below.  
-          
+If you use the container to build the environment, please directly using the the account and the password to log in to the Pulsar manager website as below.
```

Comment:
If you are deploying Pulsar Manager 0.1.0 using the released container, you can log in the Pulsar Manager UI using the following credentials.